### ISSUE TITLE
crash when user exit from activity

### ISSUE THREAD

**Issue by mypplication** [SourcePointUSA/android-cmp-app#48]
if the user exit the activity that contain CMP (and we call ConsentLib.onDestroy() in onDestroy activity lifecycle), we can have a crash
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void dik.setLayoutParams(android.view.ViewGroup$LayoutParams)' on a null object reference
       at com.sourcepoint.cmplibrary.ConsentWebView.display + 262(ConsentWebView.java:262)
       at com.sourcepoint.cmplibrary.ConsentLib$8.run + 487(ConsentLib.java:487)
       at android.os.Handler.handleCallback + 907(Handler.java:907)
       at android.os.Handler.dispatchMessage + 105(Handler.java:105)
       at android.os.Looper.loop + 216(Looper.java:216)
       at android.app.ActivityThread.main + 7593(ActivityThread.java:7593)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run + 524(RuntimeInit.java:524)
       at com.android.internal.os.ZygoteInit.main + 987(ZygoteInit.java:987)
```

**Comment by mypplication** [comment]
hello here !
@andresilveirah This bug is very important for us. Can you fix this issue this day ?

**Comment by andresilveirah** [comment]
Hi @mypplication Pauland my apologies for not seeing this earlier. While I can't promise this will be released today, but I guarantee I'll start working on it right away.

**Comment by andresilveirah** [comment]
@mypplication can you try the version `2.4.0` and let me know if the issue persist for you?

**Comment by andresilveirah** [comment]
@mypplication just released `2.4.1` with a slightly modified code from your PR. Please give it another try and let me know if it worked for you.

**Comment by mypplication** [comment]
@andresilveirah always the same issue in 2.4.1 :-(

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.webkit.WebView.setLayoutParams(android.view.ViewGroup$LayoutParams)' on a null object reference
       at com.sourcepoint.cmplibrary.ConsentWebView.display + 282(ConsentWebView.java:282)
       at com.sourcepoint.cmplibrary.ConsentLib$8.run + 490(ConsentLib.java:490)
       at android.os.Handler.handleCallback + 751(Handler.java:751)
       at android.os.Handler.dispatchMessage + 95(Handler.java:95)
       at android.os.Looper.loop + 154(Looper.java:154)
       at android.app.ActivityThread.main + 6119(ActivityThread.java:6119)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run + 886(ZygoteInit.java:886)
       at com.android.internal.os.ZygoteInit.main + 776(ZygoteInit.java:776)
```

**Comment by andresilveirah** [comment]
Hi @mypplication that's not great... I believe the `display` method is being called after the `destroy` has been called. Let me have another look.

**Comment by andresilveirah** [comment]
@mypplication just released version `2.4.2` indeed there was missing a null check when calling `webview.display()` (which btw, you had fixed in the PR you opened a couple of days ago).

**Comment by andresilveirah** [comment]
@mypplication I'll close this issue, I hope it has been fixed for you.
